examples/weather_report/README.md

```diff
@@ -0,0 +1,33 @@
+# rust-webassembly-weather-report
```

Comment:
Could this be written similar to [other example READMEs](https://github.com/rustwasm/wasm-bindgen/blob/master/examples/closures/README.md) to link to the guide docs and online demo?

Reply:
Yes sure. Let me work on it.